rex_lee.md

```diff
@@ -1,18 +1,14 @@
-## [W38] 09/26 - 09/30 ##
+## [W39] 10/02 - 10/06 ##
 
 ### This week ###
 - Bug 950058 - Split each download item so that all of the right part of it activates the action
-  - On the way of 3rd review.
+  - On the way of 4th review.
```

Comment:
@Rex, per comment 52 it seems you were blocked by something and didn't update for a while. You should talk to your lead or Sean to speed up your progress. Thanks.